Slim/Plugin/Podcast/Plugin.pm

```diff
@@ -82,31 +103,54 @@ sub initPlugin {
 
 sub shutdownPlugin {
 	my @played = values %recentlyPlayed;
+
 	$prefs->set('recent', \@played);
 }
 
+sub updateRecentlyPlayed {
+	my ($class, $client, $song) = @_;
+	my ($url) = unwrap($song->originUrl);
+
+	$recentlyPlayed{$url} ||= { 
+			url      => $url,
+			title    => $song->track->title,
+			cover    => Slim::Player::ProtocolHandlers->iconForURL($url, $client),
```

Comment:
You could use the podcast's cover here.

Reply:
I've seen (I think) a few podcast have a different image per episode

Reply:
Oh, I think I misread this line: I thought it was `pluginDataForIcon()`... never mind.